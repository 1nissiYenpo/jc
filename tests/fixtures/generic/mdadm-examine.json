{"device":"/dev/sdb1","magic":"a92b4efc","version":"1.1","feature_map":"0x1","array_uuid":"85c5b164:d58a5ada:14f5fe07:d642e843","name":"virttest:0","creation_time":"Tue Apr 13 23:22:16 2010","raid_level":"raid1","raid_devices":2,"avail_dev_size":"11721041656 sectors (5.46 TiB 6.00 TB)","array_size":"5860520828 KiB (5.46 TiB 6.00 TB)","data_offset":264,"super_offset":0,"unused_space":"before=80 sectors, after=0 sectors","state":"clean","device_uuid":"813162e5:2e865efe:02ba5570:7003165c","internal_bitmap":"8 sectors from superblock","update_time":"Tue Jul 26 20:16:31 2022","bad_block_log":"512 entries available at offset 72 sectors","checksum":"f141a577 - correct","events":2193679,"device_role":"Active device 0","array_state":"AA ('A' == active, '.' == missing, 'R' == replacing)","array_size_num":5860520828,"avail_dev_size_num":11721041656,"unused_space_before":80,"unused_space_after":0,"name_val":"virttest:0","checksum_val":"f141a577","checksum_state":"correct","state_list":["clean"],"array_state_list":["active","active"],"creation_time_epoch":1271226136,"update_time_epoch":1658891791}
