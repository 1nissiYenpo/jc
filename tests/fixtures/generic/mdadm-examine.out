/dev/sdb1:
          Magic : a92b4efc
        Version : 1.1
    Feature Map : 0x1
     Array UUID : 85c5b164:d58a5ada:14f5fe07:d642e843
           Name : virttest:0
  Creation Time : Tue Apr 13 23:22:16 2010
     Raid Level : raid1
   Raid Devices : 2

 Avail Dev Size : 11721041656 sectors (5.46 TiB 6.00 TB)
     Array Size : 5860520828 KiB (5.46 TiB 6.00 TB)
    Data Offset : 264 sectors
   Super Offset : 0 sectors
   Unused Space : before=80 sectors, after=0 sectors
          State : clean
    Device UUID : 813162e5:2e865efe:02ba5570:7003165c

Internal Bitmap : 8 sectors from superblock
    Update Time : Tue Jul 26 20:16:31 2022
  Bad Block Log : 512 entries available at offset 72 sectors
       Checksum : f141a577 - correct
         Events : 2193679


   Device Role : Active device 0
   Array State : AA ('A' == active, '.' == missing, 'R' == replacing)
